truss/cli/cli.py

```diff
@@ -414,6 +414,18 @@ def predict(
         "after deploy completes."
     ),
 )
+@click.option(
+    "--keep-previous-production-settings",
```

Comment:
the name is a mouthful. I'm open to a different flag name if anyone has better ideas.

Reply:
hm -- my concern with this name is that it makes it sound like we're keeping the previous production settings for **subsequent deployments**, which is typically what we actually want.

What if instead of orienting this around "keep-settings" we framed it as preserving the previous deployment.

ie: `--preserve-previous-deployment` or `--preserve-previous-production-deployment`. This makes it clear that if you don't want to mess with the settings of the previous deployment.

Reply:
Yeah those are good points. I wasn't sure about the name. I think `--preserve-previous-production-deployment` as it's really clear what is going to happen. I'm ok with `--preserve-previous-deployment` if we think that `--preserve-previous-production-deployment` is too long for a cli flag